templates/coderd.yaml

```diff
@@ -183,6 +181,12 @@ spec:
               value: {{ .Values.coderd.proxy.https | quote }}
             - name: no_proxy
               value: {{ .Values.coderd.proxy.exempt | quote }}
+            - name: TRUST_PROXY_IP
+              value: {{ .Values.coderd.trustProxyIP | quote }}
+            - name: PROXY_TRUSTED_ORIGINS
+              value: {{ join "," .Values.coderd.reverseProxy.trustedOrigins | quote }}
+            - name: PROXY_TRUSTED_HEADERS
+              value: {{ join "," .Values.coderd.reverseProxy.headers | quote }}
```

Comment:
Looks reasonable to me - but I'm not a helm expert. Glad you have the test cases, too!